.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "boilerplates"]
```

Comment:
Could you please test to make sure when installed through package control, it downloads the submodules?

Reply:
I didn't know how to do it. Can you tell me how to do it? Is it only a regular zip file with specific extension in `Installed Packages` folder?

Reply:
I removed submodule (https://github.com/kevinxucs/Sublime-Gitignore/pull/8/commits/497b28d1e9bc05e961dcafeb6add83c7d7e63b00) and added subtree (https://github.com/kevinxucs/Sublime-Gitignore/pull/8/commits/7980d1f691cbb55533cb348388d6ffdfe99e1b70). There will be no auto update as I wished to be, but the [boilerplates](https://github.com/kevinxucs/Sublime-Gitignore/tree/master/boilerplates) is even with the [source repo](https://github.com/github/gitignore) now, containing the full history and having the same structure. And there is some more improvements in this branch, e.g. https://github.com/kevinxucs/Sublime-Gitignore/pull/8/commits/5b35d778d2f4d21148016bca080029af7953d8ab.

I don't now there's any reason to use subtree instead of the old structure or no. I prefer the new one personally. But there is also https://github.com/kevinxucs/Sublime-Gitignore/pull/7 and it's up to you to decide.